jenkins/debian/debos/scripts/crush.sh

```diff
@@ -6,151 +6,181 @@ set -e
 # IMPORTANT: The Debian system is not longer functional at this point,
 # for example, apt and dpkg will stop working
 
-UNNEEDED_PACKAGES="apt libapt-pkg5.0 "\
-"ncurses-bin ncurses-base libncursesw5 libncurses5 "\
-"perl-base "\
-"debconf libdebconfclient0 "\
-"libfdisk1 "\
-"insserv "\
-"init-system-helpers "\
-"cpio "\
-"passwd "\
-"libsemanage1 libsemanage-common "\
-"libsepol1 "\
-"gzip "\
-"gnupg "\
-"gpgv "\
-"hostname "\
-"adduser "\
-"debian-archive-keyring "\
 
+filesystem()  {
+   UNNEEDED_PACKAGES="libfdisk1 cpio hostname gzip "
 
-# Removing unneeded packages
-for PACKAGE in ${UNNEEDED_PACKAGES}
-do
-	echo "Forcing removal of ${PACKAGE}"
-	if ! dpkg --purge --force-remove-essential --force-depends "${PACKAGE}"
-	then
-		echo "WARNING: ${PACKAGE} isn't installed"
-	fi
-done
+   remove_packages "${UNNEEDED_PACKAGES}"
+
+   # Partition and file system tools
+   rm -f usr/sbin/*fdisk
+   rm -f usr/sbin/mkfs*
+   rm -f usr/sbin/fsck*
+
+   # No need for fuse
+   find usr etc -name '*fuse*' -prune -exec rm -r {} \;
+
+   rm -f usr/bin/hostnamectl
```

Comment:
`hostnamectl` is not a file system utility.  Maybe remove it as part of `systemd()`?